### PR TITLE
Refactor files upload logic to do not save locally

### PR DESCRIPTION
**Pull request by renozheng9**

### CODE REVIEW

**Comment by Xuanwo** [comment]
What does `secondary development` mean? And why change files' mode from `644` to `755` ?

**Comment by Xuanwo** [comment]
Update your PR title to reflact what you want to change for this repo, and do not touch other files.

**Comment by renozheng9** [comment]
> What does `secondary development` mean? And why change files' mode from `644` to `755` ?

The old plugin cannot meet some demand and some modules and functions won't be no longer needed, the reason why files' mode has been changed is my operation fault, I'm sorry...

File: languages/wp-qingstor-zh_CN.po [hunk 1]
```diff
@@ -1,11 +1,10 @@
-# Copyright (C) 2017 WP-QingStor
+# Copyright (C) 2019 WP-QingStor
 # This file is distributed under the same license as the WP-QingStor package.
 msgid ""
 msgstr ""
-"Project-Id-Version: WP-QingStor 0.3.4\n"
-"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/wp-qingstor\n"
-"POT-Creation-Date: 2017-05-17 04:23:19+00:00\n"
-"PO-Revision-Date: 2017-MO-DA HO:MI+ZONE\n"
+"Project-Id-Version: WP-QingStor 0.1.0\n"
```
Comment (Xuanwo) [inline languages/wp-qingstor-zh_CN.po]: Bump this version please.

Comment (Xuanwo) [reply languages/wp-qingstor-zh_CN.po]: `wp-qingstor-zh_CN.mo` should be removed?

File: languages/wp-qingstor.pot [hunk 2]
```diff
@@ -1,14 +1,13 @@
-# Copyright (C) 2017 WP-QingStor
+# Copyright (C) 2019 WP-QingStor
 # This file is distributed under the same license as the WP-QingStor package.
 msgid ""
 msgstr ""
-"Project-Id-Version: WP-QingStor 0.3.4\n"
-"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/wp-qingstor\n"
-"POT-Creation-Date: 2017-05-17 04:23:19+00:00\n"
+"Project-Id-Version: WP-QingStor 0.1.0\n"
```
Comment (Xuanwo) [inline languages/wp-qingstor.pot]: Same here.

File: php/qingstor-menu.php [hunk 3]
```diff
@@ -10,78 +10,46 @@ function qingstor_settings_page()
...
-        if ($qingstor_error == QS_REQUEST_OK) {
-            $qingstor_error = $qingstor_error_save;
-        }
     }
 
     $qingstor_upload_types  = $options['upload_types'];
     $qingstor_upload_prefix = $options['upload_prefix'];
-    $qingstor_backup_prefix = $options['backup_prefix'];
-    $qingstor_mail          = $options['mailaddr'];
-    $qingstor_nbackup       = $options['backup_num'];
```
Comment (Xuanwo) [inline php/qingstor-menu.php:79]: Why remove the feature of database backup ?

File: qingstor.php [hunk 4]
```diff
@@ -1,26 +1,24 @@
 <?php
 /*
 Plugin Name: WP-QingStor
-Plugin URI:  https://github.com/yungkcx/QingStor-Plugin-for-WordPress
-Description: QingStor Plugin for WordPress. The Backup function requires zip and mysqldump program.
+Plugin URI:  https://github.com/yunify/wp-qingstor
+Description: QingStor Plugin for WordPress, support auto sync Media Library and delete local files. This project is based on the secondary development of yunify/wp-qingstor.
```
Comment (Xuanwo) [inline qingstor.php]: Remove `This project is based on the secondary development of yunify/wp-qingstor` please.

File: qingstor.php [hunk 5]
```diff
@@ -1,26 +1,24 @@
...
 Text Domain: wp-qingstor
 Domain Path: languages/
-Version:     0.3.4
-Author:      yungkcx
-Author URI:  http://yungkcx.github.io
+Version:     0.4.0
+Author:      yungkcx, maydusa
```
Comment (Xuanwo) [inline qingstor.php:9]: Add your name here.

File: readme.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# wp-qingstor
```
Comment (Xuanwo) [inline readme.md:1]: Filename should be `README.md`

Comment (renozheng9) [reply readme.md:1]: Modified already.

File: qingstor.php (hunk 5 above)
Comment (renozheng9) [reply qingstor.php:9]: Maydusa is my name.

File: qingstor.php (hunk 4 above)
Comment (renozheng9) [reply qingstor.php]: Remove already.

File: php/qingstor-menu.php (hunk 3 above)
Comment (renozheng9) [reply php/qingstor-menu.php:79]: Files are no longer uploaded locally, so the backup module cannot work any more.

File: languages/wp-qingstor.pot (hunk 2 above)
Comment (renozheng9) [reply languages/wp-qingstor.pot]: Modified already.

File: languages/wp-qingstor-zh_CN.po (hunk 1 above)
Comment (renozheng9) [reply languages/wp-qingstor-zh_CN.po]: Modified the version already. Wp-qingstor-zh_CN.mo is needed.

File: readme.md (hunk 6 above)
Comment (Xuanwo) [reply readme.md:1]: `README.md` in uppercase instead of `readme.md`

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**